packages/insomnia-app/app/ui/components/settings/plugins.js

```diff
@@ -267,7 +278,17 @@ class Plugins extends React.PureComponent<Props, State> {
             <button className="pull-right icon" onClick={this._handleClearError}>
               <i className="fa fa-times" />
             </button>
-            <div className="selectable force-pre-wrap">{error}</div>
+            <div className="selectable force-pre-wrap">
+              <b>{installPluginErrMsg}</b>
+              {`\n\nThere may be an issue with the plugin itself, as a note you can discover and install plugins from the `}
+              <a href="https://insomnia.rest/plugins/">Plugin Hub.</a>
+              <details>
+                <summary>Additionl Information</summary>
+                <pre className="pad-top-sm force-wrap selectable">
+                  <code>{error.stack || error}</code>
```

Comment:
I suggest changing this to just show `error.message`, which was the previous behavior (where `error.message` was added to state and rendered on the UI).

```suggestion
                  <code>{error.message}</code>
```